packages/plugins/braintree/src/braintree/index.ts

```diff
@@ -97,30 +106,23 @@ class Braintree {
         return (await this.gateway.plan.all()).plans;
     };
 
-    async createSubscription(planId: string, customerId: string, paymentToken: string, discounts: Discount[]): Promise<ValidatedResponse<Subscription> | {success: boolean, message: string}> {
+    async createSubscription(planId: string, customerId: string, paymentNonce: string): Promise<ValidatedResponse<Subscription> | {success: boolean, message: string}> {
         if (await this.isAlreadySubscribed(planId, customerId)) {
             return {
                 success: false,
-                message: 'User is already subscribed',
+                message: 'Risulti già iscritto a questo piano',
```

Comment:
We should avoid localized message for multi-purpose library.
Let's create an error enum maybe, so the project using this lib can decide what to do with the error.
```suggestion
                error: BRAINTREE_ERRORS: USER_ALREADY_SUBSCRIBED_TO_PLAN,
```